tests/unit/test_prometheus_schema_contract.py

```
@@ -0,0 +1,320 @@
+        configured_total = (
+            state_counts["configured_unchecked"]
+            + state_counts["configured_checked_invalid"]
+            + state_counts["configured_checked_valid"]
+        )
+        checked_total = (
+            state_counts["configured_checked_invalid"]
+            + state_counts["configured_checked_valid"]
+        )
+        success_ratio = state_counts["configured_checked_valid"] / configured_total
+        checked_ratio = checked_total / configured_total
+        self.assertGreaterEqual(success_ratio, 0.0)
```

Ruff (0.15.7)

[warning] Use a regular `assert` instead of unittest-style `assertGreaterEqual`

Replace `assertGreaterEqual(...)` with `assert ...`

(PT009)